scripts/prepare_automatic_backups.py

```diff
@@ -44,6 +53,12 @@ def generate_backup_url(cloud_storage_bucket_name, module_class_names):
 
 
 def update_cron_dict(cron_dict):
+    """Update yaml file content by updating argument passed content which is
```

Comment:
From this docstring I don't understand what the function is doing. I think it needs rewriting to be clearer.

Reply:
done